src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -180,4 +180,10 @@
   <data name="SARIF1014_UriBaseIdRequiresRelativeUri" xml:space="preserve">
     <value>If a fileLocation object contains a "uriBaseId" property, the value of the "uri" property must be a relative URI reference.</value>
   </data>
+  <data name="SARIF1015_Default" xml:space="preserve">
+    <value>{0}: The SARIF specification requires the value of this property to be an absolute URI, but "{1}" is a relative URI reference.</value>
+  </data>
+  <data name="SARIF1015_UriMustBeAbsolute" xml:space="preserve">
+    <value>Certain URIs in the SARIF specification are required to be absolute.</value>
```

Comment:
>Certain [](start = 11, length = 7)

Likewise here, just remove the words "in the SARIF specification".

Reply:
yes

---
In reply to: [211696777](https://github.com/Microsoft/sarif-sdk/pull/1040#discussion_r211696777) [](ancestors = 211696777)